docs/faq.md

```diff
@@ -72,11 +72,15 @@ However, if you meant an activity like this:
 
 You can use either of the following
 
-* The overload for `DiscordClient#ConnectAsync()` which accepts a `Discord​Activity`.
-* `DiscordClient#UpdateStatusAsync()` at any point after `Ready` has been fired.
+* The overload for @DSharpPlus.DiscordClient.ConnectAsync(DiscordActivity,System.Nullable{UserStatus},System.Nullable{DateTimeOffset}) which accepts a @DSharpPlus.Entities.DiscordActivity.
+* @DSharpPlus.DiscordClient.ConnectAsync(DiscordActivity,System.Nullable{UserStatus},System.Nullable{DateTimeOffset}) at any point after `Ready` has been fired.
```

Comment:
Is this meant to be `UpdateStatusAsync()` instead? If so, also add a link to the sharded client method.

Reply:
Ill fix this tonighy, i copied and pasted the wrong xref